STYLE_GUIDE.md

```diff
@@ -0,0 +1,110 @@
+# ComplyTime Organization Style Guide
+
+This document serves as the central source of truth for engineering standards, contribution workflows, and architectural principles for the ComplyTime organization. All contributors and maintainers are expected to adhere to these guidelines to ensure consistency, quality, and compliance.
+
+---
+
+## 1. High-Level Engineering Principles
+
+These principles guide *how* we think about problems and *why* we make specific technical decisions. We value long-term sustainability over short-term speed.
+
+### Core Value: Maintainability & Robustness
+
+**1. Single Source of Truth (Centralized Constants)**
+* **Principle:** Define a value once, use it everywhere. Never hardcode the same string or number in multiple places.
```

Comment:
I feel like this could be less absolute to allow developers to use their judgement
```suggestion
* **Principle:** Centralize values that are used in multiple places or may change over time.
```

Reply:
Updated. Thanks